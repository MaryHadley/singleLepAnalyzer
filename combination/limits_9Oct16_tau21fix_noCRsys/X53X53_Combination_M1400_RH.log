000 (1297360.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297360.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.240:42701>
...
006 (1297360.000.000) 10/10 19:27:26 Image size of job updated: 11336
	2  -  MemoryUsage of job (MB)
	1144  -  ResidentSetSize of job (KB)
...
006 (1297360.000.000) 10/10 19:32:26 Image size of job updated: 282744
	158  -  MemoryUsage of job (MB)
	161624  -  ResidentSetSize of job (KB)
...
006 (1297360.000.000) 10/10 20:17:31 Image size of job updated: 283824
	158  -  MemoryUsage of job (MB)
	161624  -  ResidentSetSize of job (KB)
...
005 (1297360.000.000) 10/10 20:21:58 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:54:12, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:54:12, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1631  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1631  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1095        1   3824812
	   Memory (MB)          :      158     3072      3072
...
