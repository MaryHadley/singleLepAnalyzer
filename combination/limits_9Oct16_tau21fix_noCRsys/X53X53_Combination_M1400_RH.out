/var/lib/condor/execute/dir_9700
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_RH/quant-toys:0-sig-780dd10f84.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_RH/quant-data-sig-1dc96620c6.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_RH
