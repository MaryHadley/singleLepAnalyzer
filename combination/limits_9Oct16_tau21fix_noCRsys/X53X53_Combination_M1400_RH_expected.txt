# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.069627   0.034422    0.14374  0.047989    0.10163
